frontend/cypress/integration/all.spec.js

```diff
@@ -4,11 +4,12 @@ import { containersTests } from "./containers.spec"
 import { individualsTests } from "./individuals.spec"
 import { samplesTests } from "./samples.spec"
 import { protocolsTests } from "./protocols.spec"
+import { paginationTests } from "./pagination.spec"
 
 // Tests
 context('All tests', () => {
```

Comment:
A suggestion: since FreezeMan is growing, and we're adding more and more tests, I'd be great to be able to run some of the independent tests on their own (pagination test is the perfect example of that, because this test does not rely on other tests for their data, so it can be run alone)